### DISCUSSION TITLE
How to place the mat-card with another solution than CSS?

### DISCUSSION THREAD

**Discussion by mashio26** [hakimio/ngx-ui-tour#108] [Q&A] [answered]
Hello, I'm using material design and I wont to place the mat-card but if I used the solution of CSS i will move all my form.
There is a another solution ?
![image](https://user-images.githubusercontent.com/84849078/220309860-171d10c7-e3b3-4f82-bb5a-8665090d1e93.png)

**Comment by hakimio** [comment, marked as answer]
There is [a placement config](https://hakimio.github.io/ngx-ui-tour/md-menu/API#:~:text=placement,the%20anchor%20(optional).) you can use to adjust popover position.

**Reply by mashio26** [reply]
Thanks ! But i will not use enableBackdrop on a modal.

**Reply by hakimio** [reply]
You can try to set `backdropContainer` if backdrop stacking is not correct and it appears on top of the tour step.
